test/RuleTests.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-env node, jasmine, mocha */
+import {
+  configure,
+  // render,
+} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { expect } from 'chai';
+// Utils
+import {
+  mountHGComponent,
+  removeHGComponent,
```

Comment:
```suggestion
  // removeHGComponent,
```

Again just a ESLint issue causing trouble with Travis